db/init/00-database.sql

```diff
@@ -1,30 +1,59 @@
 CREATE DATABASE clinical_trials_status;
-
 \connect clinical_trials_status;
 
 CREATE SCHEMA trials_status_schema;
 
-CREATE TABLE trials_status_schema.institutions (
-    id SERIAL PRIMARY KEY,
-    org_name TEXT,
-    org_type TEXT,
-    created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    updated_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+CREATE TABLE trials_status_schema.organizations (
+  id SERIAL UNIQUE,
+  org_full_name TEXT,
+  org_class TEXT,
+  total_count BIGINT DEFAULT 0,
+  should_have_results_count BIGINT,
+  late_count BIGINT DEFAULT 0,
+  missing_count BIGINT DEFAULT 0,
+  on_time_count BIGINT DEFAULT 0,
+  late_frac FLOAT DEFAULT 0.5,
```

Comment:
Is there a reason this shouldn't be 0 or None?

Reply:
Yeah, that's better. Switched it to `NULL`